### DISCUSSION TITLE
Support automaton

### DISCUSSION THREAD

**Discussion by kumori123** [kind2-mc/kind2#921] [Q&A] [answered]
Hi，I was wondering if KIND2 support automaton now or in future? 
Thank you for your reply!

**Comment by daniel-larraz** [comment, marked as answer]
We discontinued the support for automata in version 1.6.0. You can find further information in this [comment](https://github.com/kind2-mc/kind2/pull/835#issuecomment-1134876679).

**Reply by kumori123** [reply]
Thank you for your reply!

**Reply by kumori123** [reply]
Is there any version of KIND2 which still supports statemachine?

**Reply by kumori123** [reply]
Oh, I tried 1.5.1 and it works.
